spec/controllers/api/documents_controller_spec.rb

```diff
@@ -26,28 +28,39 @@
       let(:params) { {} }
 
       it 'returns all documents' do
-        expect(meta_data['results']).to eq 4
-        expect(documents.count).to eq 4
+        expect(meta_data['results']).to eq 6
+        expect(documents.count).to eq 6
       end
     end
 
     context 'when documents of type insight are requested' do
       let(:params) { { document_type: 'insight' } }
       it 'returns all insight documents' do
-        expect(meta_data['results']).to eq 3
-        expect(documents.count).to eq 3
+        expect(meta_data['results']).to eq 5
+        expect(documents.count).to eq 5
       end
     end
   end
 
   describe 'keyword search' do
     context 'when the document_type is specified' do
       context 'when a keyword is provided' do
-        let(:params) { { document_type: 'insight', keyword: 'pension' } }
+        context 'and the keyword is found in the content' do
+          let(:params) { { document_type: 'insight', keyword: 'pension' } }
 
-        it 'returns an array of documents which contain the keyword' do
-          expect(meta_data['results']).to eq 1
-          expect(documents.count).to eq 1
+          it 'returns an array of documents which contain the keyword' do
+            expect(meta_data['results']).to eq 1
+            expect(documents.count).to eq 1
+          end
+        end
+
+        context 'and the keyword is found in the title' do
+          let(:params) { { document_type: 'insight', keyword: 'annuity' } }
+
+          it 'returns an array of documents which contain the keyword' do
+            expect(meta_data['results']).to eq 2
+            expect(documents.count).to eq 2
+          end
```

Comment:
Can we assert about the document info instead of using documents.count? For the page type makes sense but for search by keyword would be good to assert that the document that you search is the document that is returned by the API.